client/beefy/src/aux_schema.rs

```diff
@@ -72,6 +73,118 @@ where
 	Ok(None)
 }
 
+mod v1 {
```

Comment:
Since old aux-schema version has never been run in production I propose to actually drop this conversion and support for old version.